packages/core/src/utils/slang/trivia.ts

```diff
@@ -0,0 +1,20 @@
+import { TerminalKind } from '@nomicfoundation/slang/kinds';
+import { Node, TerminalNode } from '@nomicfoundation/slang/cst';
+
+/**
+ * Returns true if the node is a trivia terminal (whitespace or comment or NatSpec)
+ */
+export function isTrivia(node: Node) {
```

Comment:
Have you suggested they add this to the Slang API? I see it's defined for Rust but not exposed in the npm package.

Reply:
Yes, this was suggested for the npm package.

Reply:
Slang recently published `v0.18.2` with a new utility `TerminalKindExtensions.isTrivia()`. Thanks for suggesting!